### PR TITLE
Home test plan

### PR DESCRIPTION
**Pull request by martypowell**
Addresses the index page test plan

https://github.com/baltimorecounty/BCPL-assets/wiki/Index-Page-Test-Plan

### CODE REVIEW

File: cypress/integration/home-page.spec.js [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+            cy
+                .get(constants.selectors.blogPost)
+                .first()
+                .find('a')
+                .click();
+
+            assertLocationChanged();
+        });
+
+        it('should navigate a list of blog posts when view the blog is selecoted', () => {
```
Comment (jsmith176) [inline cypress/integration/home-page.spec.js]: Change _selecoted_ to selected

File: cypress/integration/home-page.spec.js [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+            getActiveTab().contains('Fiction');
+        });
+
+        it('should allow users to move through titles using the previous arrow in the control', () => {
+            getVisibleTitles(setVisibleTitles);
+
+            getCarouselArrow()
+                .first()
+                .click()
+                .wait(500);
```
Comment (jsmith176) [inline cypress/integration/home-page.spec.js]: I am not sure about this but have you tried leaving off `.wait(500)`? The [docs](https://docs.cypress.io/guides/core-concepts/interacting-with-elements.html#Animations) say that cypress automatically waits for animations to finish.

File: cypress/integration/home-page.spec.js (hunk 1 above)
Comment (martypowell) [reply cypress/integration/home-page.spec.js]: done

File: cypress/integration/home-page.spec.js (hunk 2 above)
Comment (martypowell) [reply cypress/integration/home-page.spec.js]: per our chat, going to leave this, because it's checking a different element.

**Pull request merged**